INSTALLATION.md

```diff
@@ -0,0 +1,91 @@
+# Atomkraft Installation Instructions
+
+Installing Atomkraft is as easy as `pip install atomkraft`.
+However, atomkraft depends on some software:
+your mileage may vary based on how much of that software you already have installed.
+In the following sections we describe how to install and what features are supported by different dependencies.
+If you are looking for a copy&paste solution to get Atomkraft up and running as quickly as possible,
+check the [TL;DR section](#tldr).
+
+## Installing Atomkraft
+
+Installing Atomkraft requires [Python](https://www.python.org) 3.10.
+
+If you don't have Python 3.10 already installed, please consult the [installation instructions](https://realpython.com/installing-python/) for different installation options.
```

Comment:
Why do we reference to `realpython` blog here? Why not official instructions at https://www.python.org/downloads?